src/libstd/os/raw/schar.md

```diff
@@ -0,0 +1,6 @@
+Equivalent to C's `signed char` type.
+
+This type will almost always be [`i8`], but its size is technically equal to the size of a C [`char`], which isn't very clear-cut.
```

Comment:
The use of "isn't very clear-cut" throughout the number docs feels awkward. Is there *any* guidance we can paraphrase here? If not, i'd rather we use a phrase like "which is left under-specified" to make it feel marginally less wishy-washy.

Reply:
Fair; I was trying to avoid duplicating the docs too much, although it's mostly wishy-washy because C's definitions are incredibly wishy-washy.

I'll probably replace it with "which does not have a strict definition like Rust does" or something a bit more eloquent.

Reply:
I don't see how C doesn't have a strict definition of size of `signed char` 😄. In C a `char`'s size is defined to be 1 byte, always (although 1 byte is not defined to be 8 bits) (C11 §6.5.3.4/4). 

Unless Rust is going to support non-8-bit-byte system, a `signed char` is always an `i8`.